backend/src/Controller/PlayerController.php

```diff
@@ -14,7 +15,7 @@
 use function App\createErrorResponse;
 
 /**
- * @Route("/player", name="player")
+ * @Route("/api/player", name="player")
```

Comment:
I also made the proper routing for this controller that should be merged anytime soon